src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -412,6 +413,11 @@
       }
     }
 
+    .pf-m-split-toggle-button {
```

Comment:
As discussed offline:
* rename to `.pf-m-text`
* Move to the block below, and add `&` before the selector, so it's `&.pf-m-text` https://github.com/patternfly/patternfly/blob/251039c193ba2c138bb253f3da2a3237478454eb/src/patternfly/components/MenuToggle/menu-toggle.scss#L568-L579